<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCubicEquation</class>
 <widget class="QWidget" name="FormCubicEquation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>566</width>
    <height>170</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>560</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Решение кубических уравнений</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>A*X^3+B*X^2+C*X+D = 0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxA"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxB"/>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxC"/>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxD"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="4">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelX1">
           <property name="text">
            <string>X1 = </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelX2">
           <property name="text">
            <string>X2 = </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelX3">
           <property name="text">
            <string>X3 = </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Количество десятичных знаков</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBoxPrec"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonCalculate">
           <property name="text">
            <string>Решить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonExit">
           <property name="text">
            <string>Выйти</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
